public/app/plugins/datasource/mssql/datasource.test.ts

```diff
@@ -80,6 +80,26 @@ describe('MSSQLDatasource', () => {
     });
   });
 
+  describe('When performing metricFindQuery and backendSrv returns undefined dataframe', () => {
```

Comment:
Nitpick, is this testing for returning "undefined" dataframes or rather "empty" dataframes?